ingestor.js

```diff
@@ -0,0 +1,80 @@
+// TODOs
+// We should add the unzipper somewhere in here as well
+// Find out if we can get the relative file path instead of absolute
+// Next step after we've recursed over the archive would be to insert all the parsed data into the data base, aligning data with our mongoDB data model
+
+const fs = require('fs');
+const path = require('path');
+const { processFileController } = require('./utils.js');
+
+// Local State
+let state = {};
```

Comment:
After file walker is done, we are going to filter the parsed json data and send to the db. I thought it would be nice to have access to the 'state' variable to do so. I'll change filewalker to return that instead.

Reply:
It won't store that information unless it's in a class. When you import the `state` from this file you're only importing an empty Object.